psi4/driver/p4util/python_helpers.py

```diff
@@ -185,6 +185,15 @@ def set_module_options(module, options_dict):
 def pcm_helper(block):
     """Passes multiline string *block* to PCMSolver parser."""
 
+    # Setup unique scratch directory and move in, as done for other add-ons
+    psioh = core.IOManager.shared_object()
+    psio = core.IO.shared_object()
+    os.chdir(psioh.get_default_path())
+    pcm_tmpdir = 'psi.' + str(os.getpid()) + '.' + psio.get_default_namespace() + \
+        'pcmsolver.' + str(uuid.uuid4())[:8]
```

Comment:
The above sequence of commands is probably a candidate for a util function. Adding to developer development list.